### ISSUE TITLE
Failed Payment with the latest sdk

### ISSUE THREAD

**Issue by PembaTamang** [razorpay/razorpay-android-sample-app#207]
I am using the latest sdk 1.6.5
with the same kotlin code from this repo.

The first error when I start the payment.

` E/com.razorpay.checkout: Webview JS Error: Uncaught TypeError: window.getDeviceDetails is not a function`

after I leave the activity an IntentReceiver is leaked

```
Webview JS Error: Uncaught ReferenceError: otpPermissionCallback is not defined
2021-02-25 16:03:50.553 20789-20789/com.pembatamang.ecommercedemo E/ActivityThread: Activity com.razorpay.CheckoutActivity has leaked IntentReceiver com.razorpay.CheckoutPresenterImpl$1@b732713 that was originally registered here. Are you missing a call to unregisterReceiver()?
    android.app.IntentReceiverLeaked: Activity com.razorpay.CheckoutActivity has leaked IntentReceiver com.razorpay.CheckoutPresenterImpl$1@b732713 that was originally registered here. Are you missing a call to unregisterReceiver()?
        at android.app.LoadedApk$ReceiverDispatcher.<init>(LoadedApk.java:1707)
        at android.app.LoadedApk.getReceiverDispatcher(LoadedApk.java:1438)
        at android.app.ContextImpl.registerReceiverInternal(ContextImpl.java:1577)
        at android.app.ContextImpl.registerReceiver(ContextImpl.java:1550)
        at android.content.ContextWrapper.registerReceiver(ContextWrapper.java:654)
        at com.razorpay.CheckoutPresenterImpl$8.onFinish(CheckoutPresenterImpl.java:775)
        at android.os.CountDownTimer$1.handleMessage(CountDownTimer.java:127)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:237)
        at android.app.ActivityThread.main(ActivityThread.java:8167)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)
```

**Comment by sumedht** [comment]
@PembaTamang Checking with it. Meanwhile, u can use our latest SDK version 1.6.6

**Comment by sumedht** [comment]
@PembaTamang Have fixed this in v1.6.7
